lib/Github/Model/User.php

```diff
@@ -0,0 +1,99 @@
+<?php
+namespace Github\Model;
+
+use function Makasim\Values\get_value;
+
+class User
+{
+    private $values = [];
+
+    public function getLogin()
+    {
+        return get_value($this, 'login');
+    }
+
+    public function getId()
+    {
+        return get_value($this, 'id');
+    }
+
+    public function getNodeId()
+    {
+        return get_value($this, 'node_id');
+    }
+
+    public function getAvatarUrl()
+    {
+        return get_value($this, 'avatar_url');
+    }
+
+    public function getGravatarUrl()
+    {
+        return get_value($this, 'gravatar_id');
```

Comment:
`gravatar_id` or `gravatar_url` ?

Reply:
`gravatar_id` is correct but maybe we should rename the method to match the actual property

Reply:
Yes we should do this